Makefile

```diff
@@ -27,9 +28,9 @@ frontend:
 	docker build --build-arg CONFIGURATION=local -t t4c/client/frontend -t $(LOCAL_REGISTRY_NAME):$(REGISTRY_PORT)/capella/collab/frontend frontend
 	docker push $(LOCAL_REGISTRY_NAME):$(REGISTRY_PORT)/capella/collab/frontend
 
-capella: capella-download
+capella:
```

Comment:
Is downloading capella done in the docker image?

Reply:
Yes :) 
It's a new feature of `capella-dockerimages`, so we don't need to do it manually anymore.